schemas/common/externalIdsHolder.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Contains identifiers of external entities (instance, holding)",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "instanceId": {
```

Comment:
Please rename it from "instanceId" to "inventoryInstanceId" to be able to differ instance from mod-inventory/mod-marccat

Reply:
let's leave it as instanceId for now, it reflects the 'i' subfiled of '999' field.